packages/dashboards/examples/src/components/prototypes/prototype-1/global-filtering-data.source.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from "@angular/core";
+import { DataSourceService, IFilteringOutputs, IFilters } from "@nova-ui/bits";
+
+@Injectable()
+export class GlobalFilteringDataSource extends DataSourceService<any> {
+
+    public getFilteredData(filters: IFilters): Promise<IFilteringOutputs> {
+        return Promise.resolve({} as IFilteringOutputs);
```

Comment:
The problem here is really only that I need to register filters somewhere and it can't be done without a data source. This is here just to satisfy the types and be runtime safe.